### ISSUE TITLE
Phantom selection chrome

### ISSUE THREAD

**Issue by aerwin** [maqetta/maqetta#3343]
@JonFerraiolo @jhpedemonte 

Using Chrome on Mac going to staging server... seems somewhat intermittent, but I think the scenario below is fairly reliable... 
1. New HTML file
2. Add GridX to new file
3. Create another new HTML file
4. Close new HTML file without saving
5. Close first HTML file without saving
6. See flicker of selection chrome on welcome page
7. Create new HTML file and see selection chrome in shape of GridX even though no widgets added:

![GitHub Logo](http://i.imgur.com/Q2qgB.png)

**Comment by JonFerraiolo** [comment]
@doronrosenberg @billreed63 @peller @childsb @jhpedemonte - if anyone is seeing phantom selection chrome on occasion in situations where there are no gridx widgets, please add a note to this issue. Thanks.

**Comment by aerwin** [comment]
@JonFerraiolo can't replicate this one... I tried from my local Eclipse launch script as well and don't see it there either... so I guess just a weird timing issue going to server that for moment just I am seeing...

**Comment by JonFerraiolo** [comment]
As part of my playing around with gridx scenarios, I did see the phantom selection chrome one time. it was after the messages to the console reported in #3345. I suspect that the exception is preventing the logic from completing that normally gets rid of the old selection chrome. 

I looked into whether there is an easy fix to make sure that when you switch editors, any old selection chrome gets removed, but such a change would be too risky at this point. Might cause other problems. So, I'm pushing this issue to M8. If users see this problem, just tell them sorry, hit browser refresh.

-> M8 

@jhpedemonte
